docs/rdbms/naming-convention.md

```diff
@@ -0,0 +1,134 @@
+---
+id: rdbms-naming-convention
+title: RDBMS Naming Convention
+sidebar_label: Naming Convention
+---
+
+#### The following convention should be followed for naming RDBMS:
+
+* Use `snake_case` with lowercase while naming.
+* Use uppercase for SQL keywords and built-in functions.
+* Avoid use of dots, spaces, or dashes in database, schema, table, or column names.
+
+### Database
+
+Example `dvd_rental`.
+
+```sql
+--
+-- Good
+CREATE DATABASE dvd_rental;
+
+--
+-- Bad
+CREATE DATABASE [dvd rental];
+```
+
+### Schema
+
+Schema works as a namespace. Example `raw`, `ops`, `util`.
+
+```sql
+CREATE SCHEMA util;
+```
+
+### Table
+
```

Comment:
When naming tables, you have two options – to use the singular for the table name or to use a plural. Singular table name is more preferred